modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -36,17 +36,19 @@ spec:
   managementState: Managed
   operatorLogLevel: Normal
   profileCustomizations:
-    podLifetime: 48h        <1>
-  profiles:                 <2>
+    podLifetime: 48h                                   <1>
+    thresholdPriorityClassName: my-priority-class-name <2>
+  profiles:                                            <3>
   - AffinityAndTaints
-  - TopologyAndDuplicates   <3>
+  - TopologyAndDuplicates                              <4>
   - LifecycleAndUtilization
   - EvictPodsWithLocalStorage
   - EvictPodsWithPVC
 ----
 <1> Optional: Enable a custom pod lifetime value for the `LifecycleAndUtilization` profile. Valid units are `s`, `m`, or `h`. The default pod lifetime is 24 hours.
-<2> Add one or more profiles to enable. Available profiles: `AffinityAndTaints`, `TopologyAndDuplicates`, `LifecycleAndUtilization`, `SoftTopologyAndDuplicates`, `EvictPodsWithLocalStorage`, and `EvictPodsWithPVC`.
-<3> Do not enable both `TopologyAndDuplicates` and `SoftTopologyAndDuplicates`. Enabling both results in a conflict.
+<2> Optional: Specify a priority threshold to consider pods for eviction only if their priority is lower than the specified level. Use the `thresholdPriority` field to set a numerical priority threshold or use the `thresholdPriorityClassName` field to specify a certain priority class name. If you specify a priority class name, it must already exist or the descheduler will throw an error. Do not set both `thresholdPriority` and `thresholdPriorityClassName`.
+<3> Add one or more profiles to enable. Available profiles: `AffinityAndTaints`, `TopologyAndDuplicates`, `LifecycleAndUtilization`, `SoftTopologyAndDuplicates`, `EvictPodsWithLocalStorage`, and `EvictPodsWithPVC`.
```

Comment:
This reminds me that I have been trying to figure out how to do an unordered list under a callout for one of my YAML files. Do you know if it's possible? I couldn't make it align correctly and it restarted the numbering after it :upside_down_face:

Reply:
@jeana-redhat I'm not positive. I've found that the formatting you can do with callouts is pretty limited. Sometimes it only works if you're trying to do it on the last callout and don't have to have any more following it :shrug: 

I feel like I _might_ have seen someone do unnumbered bullets under a callout and continue afterward, but it definitely wasn't me, and I don't know how to find where that might have been. Sorry!